apache/trafficcontrol
# Support queueing updates by type and profile

## What does this PR (Pull Request) do?


- [x] This PR fixes #4982  and fixes #3669


## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops
- CI tests

## What is the best way to verify this PR?


Make sure all the API/ unit tests pass.
Manual testing:
Run TO and TP locally.
Go to TP and click on a `TYPE` that is used in the server table, for example, `EDGE`.
Under the `More` dropdown button, you should be able to queue updates for this particular type and any CDN.
Do a similar test for profiles.
## If this is a bug fix, what versions of Traffic Control are affected?


- master
## The following criteria are ALL met by this PR




## Additional Information
